Describe the applyImageFit property (#109)
surveyjs/survey-pdf: docs/Options-in-Constructor.md

Obsolete.  

- [useCustomFontInHtml](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#40)  
`useCustomFontInHtml?: boolean;`  
Notes:  

> Earlier was used together with [base64Normal](base64Normal) and [base64Bold](base64Bold) to specify whether [custom fonts](https://surveyjs.io/Documentation/Pdf-Export?id=Customization-ChangeFonts#use-custom-font) (loaded throught the `base64Normal` and `base64Bold` options) should apply to texts rendered within questions of the HTML type.  

Reviewer: This sentence needs a rewrite.
You have a word missing right at the beginning: what was used together...?
Also, odd phrasing on "texts rendered within questions of the HTML type"

<a id="commercial"></a>
## Commercial license

Use the [commercial](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L42) or [haveCommercialLicense](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L43) option to specify whether you have a commercial license for  the **Survey PDF Export** library.

- [commercial](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L42)  
`commercial?: boolean;`

- [haveCommercialLicense](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L43)  
`haveCommercialLicense?: boolean;`

> Setting an option to true removes a non-commercial usage warning displayed on the top of the document.  

Reviewer: ```suggestion
Setting an option to true removes a non-commercial usage warning displayed at the top of the document.  
```

<a id="htmlRenderAs"></a>
## HTML question render mode

> Use the [htmlRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L44) option to specify how to render questions of the [HTML type](https://surveyjs.io/Documentation/Library/?id=questionhtmlmodel) into the result PDF document.

Reviewer: ```suggestion
Use the [htmlRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L44) option to specify how to render questions of the [HTML type](https://surveyjs.io/Documentation/Library/?id=questionhtmlmodel) into the resulting PDF file.
```

- [htmlRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L44)  
`htmlRenderAs?: IHTMLRenderType;`

Possible values are listed in [IHTMLRenderType](https://github.com/surveyjs/survey-pdf/blob/master/src/flat_layout/flat_html.ts#L9): 

- "auto"  

> SurveyPDF automatically selects how to render each HTML question (as a standard text or as an image) depending upon the complexity of the processed HTML markup.

Reviewer: ```suggestion
SurveyPDF automatically selects how to render each HTML question (as a standard text or as an image) depending on the complexity of the processed HTML markup.
```

> For each particular HTML question, you can override this general ("auto") behavior at the question level - by specifying a question's `renderAs` property (in a survey JSON definition). Set the property to "standard" or "image" as well (its default value is "auto").

Reviewer: ```suggestion
For each HTML question, you can override this general ("auto") behavior at the question level: specify a question's `renderAs` property (in a survey JSON definition); also set the property to "standard" or "image" (its default value is "auto").
```
This two sentences don't make sense to me. I don't really know what you are saying and how the second sentence relates to the first. Please rewrite.

<a id="matrixRenderAs"></a>
## Matrix question render mode

> Use the [matrixRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L45) option to specify how to render questions of the [matrix types](https://surveyjs.io/Examples/Library?id=questiontype-matrix) into the result PDF document.

Reviewer: ```suggestion
Use the [matrixRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L45) option to specify how to render [matrix types](https://surveyjs.io/Examples/Library?id=questiontype-matrix) questions into the resulting PDF file.
```

> For each particular matrix question, you can override this general behavior at the question level - by specifying a question's `renderAs` property (in a survey JSON definition).

Reviewer: ```suggestion
For each matrix question, you can override this general behavior at the question level: specify the question's `renderAs` property (in a survey JSON definition).
```

<a id="readonlyRenderAs"></a>
## Render mode of read-only questions

> The [readonlyRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L46) option allows you to fine tune the render of read-only questions in a PDF document being exported.

Reviewer: ```suggestion
The [readonlyRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L46) option allows you to fine tune [please replace fine tune with another word] the render of read-only questions in a PDF file that you want to export.
```

> For each particular question, you can override the default render behavior at the question level - by programmatically specifying a question's `readonlyRenderAs` custom property.

Reviewer: ```suggestion
For each question, you can override the default render behavior at the question level: programmatically specify a question's `readonlyRenderAs` custom property.
```

<a id="textFieldRenderAs"></a>
## Text input render mode in read-only questions

> Use the [textFieldRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L47) option to specify the manner in which single-line text fields display respondent answers (especially long ones) in the result PDF within questions of the following types: 

Reviewer: ```suggestion
Use the [textFieldRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L47) option to specify the manner in which single-line text fields display respondent answers (especially long ones) in the resulting PDF within questions of the following types: 
```
The last part of this sentence doesn't make sense (after fields). Rewrite.

> The [textFieldRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L47) option is only **in effect** for questions which are used **in read-only mode** due to one of the following settings:

Reviewer: ```suggestion
The [textFieldRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L47) option is only **in effect** for questions used **in read-only mode** due to one of the following settings:
```

 
>You can identify a question's current state by using the [isReadOnly](https://surveyjs.io/Documentation/Library?id=Question#isReadOnly) property.

> The [textFieldRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L47) option allows you to better present long answers to open-ended questions in a PDF document.

Reviewer: ```suggestion
The [textFieldRenderAs](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L47) option allows you to present long answers to open-ended questions as a PDF file.
```

<a id="compress"></a>
## Document compression

Use the [compress](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L48) option to specify whether to compress the  generated PDF document. A compressed document does not support using [custom fonts](https://surveyjs.io/Documentation/Pdf-Export?id=Customization-ChangeFonts#use-custom-font).

- [compress](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L48)  
`compress?: boolean;`

```js
var options = {
    compress: true
};
var survey = new SurveyPDF.SurveyPDF(json, options);
```

## Image Rendering

> [Image questions](https://surveyjs.io/Documentation/Library?id=questionimagemodel) have an [imageFit](https://surveyjs.io/Documentation/Library?id=questionimagemodel#imageFit) property that specifies how images should fit into their container. If exported images should take this property into account, enable the `applyImageFit` option. Please note that with this option, the quality of images may be lower because they pass through several conversions.

Reviewer: ```suggestion
[Image questions](https://surveyjs.io/Documentation/Library?id=questionimagemodel) have an [imageFit](https://surveyjs.io/Documentation/Library?id=questionimagemodel#imageFit) property that specifies how images should fit into their containers. If exported images should take this property into account [should take this property into account?], enable the `applyImageFit` option. Please note that with this option, the quality of images may be lower because they pass through several conversions.
```

Author: ```suggestion
[Image questions](https://surveyjs.io/Documentation/Library?id=questionimagemodel) have an [imageFit](https://surveyjs.io/Documentation/Library?id=questionimagemodel#imageFit) property that specifies how images should fit into their containers. If exported images should apply this property, enable the `applyImageFit` option. Please note that with this option, the quality of images may be lower because they pass through several conversions.
```

Reviewer: Roman, this work. :)
